docs/learn/learn-common-good-chains.md

```diff
@@ -89,7 +89,7 @@ Statemint will require the Relay Chain's "root origin", i.e. a referendum. Some
 
 ### Encointer
 
-Encointer will become Kusama's second common good parachain, which adds logic to the Relay Chain
+Encointer is a blockchain platform for self-sovereign ID and a global universal basic income. With [referendum 187](https://kusama.polkassembly.io/referendum/187) Encointer became Kusama's second common good parachain and it adds logic to the Relay Chain
```

Comment:
Encointer was registered as a common good parachain through this referendum https://kusama.polkassembly.io/referendum/158 and got upgraded to its full functionality through the linked referendum. Rephrase it accordingly if needed to reflect both referenda.

```suggestion
Encointer is a blockchain platform for self-sovereign ID and a global universal basic income. With [referendum 158](https://kusama.polkassembly.io/referendum/158) Encointer became Kusama's second common good parachain and it adds logic to the Relay Chain
```

Reply:
Yeah I think both will be helpful